action.yaml

```diff
@@ -18,6 +18,11 @@ inputs:
     description: working directory
     required: true
     default: "."
+  terraform-plan-output:
```

Comment:
won't you have to also introduce a plan-output for the drift part? Especialy for cd prod/release needs? :thinking:

Reply:
Parent action decide where plan should be stored

Reply:
:+1: for the `plan`, but what about the `drift`?

Reply:
Parsed on message variable